extensions/douban/src/search-groups.tsx

```diff
@@ -98,6 +99,11 @@ export default function Command() {
                   shortcut={{ modifiers: ["cmd", "shift"], key: "d" }}
                   onAction={() => setShowingDetail(!showingDetail)}
                 />
+                <Action.CopyToClipboard
+                  title="Copy As Markdown"
+                  content={`[${group.title}(豆瓣)](${group.url})`}
```

Comment:
At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.